mandatory/3-function-output.js

```diff
@@ -2,16 +2,21 @@
 function getRandomNumber() {
   return Math.random() * 10;
 }
+// MATH.RANDOM : to find a random number between 0 to 1. so tofind a number between 0 to 100 we need to multiple it to 100.
 
 // Add comments to explain what this function does. You're meant to use Google!
 function combine2Words(word1, word2) {
   return word1.concat(word2);
-}
+} 
+// CONCAT: to mix arrays together we can use concat. the other way is spread operator.
```

Comment:
I understand what you are saying here and this answer is correct.  🥇 

A clearer way to say it might be to say link or connect instead of mix. Mix implies the values might be "mixed up" / disordered in the new array, but they retain their original sequence order.

CONCAT: to **link** arrays together we can use concat. the other way is spread operator.